apps/bluetooth/app.js

```diff
@@ -544,7 +544,7 @@ module.exports = function (activity) {
     switch (state) {
       case protocol.DISCOVERY_STATE.ON:
         if (lastIntent !== 'implicit_disconnect' && lastIntent !== 'bluetooth_disconnect') {
-          activity.light.play(res.LIGHT.DISCOVERY_ON, {}, { shouldResume: true })
+          activity.light.play(res.LIGHT.DISCOVERY_ON, {}, { shouldResume: true, zIndex: 2 })
```

Comment:
It's better to define a well readable constant for zIndex value.

Reply:
+1